src/content-gate/editor/editor.js

```diff
@@ -36,6 +36,7 @@ const overlaySizes = [
 ];
 
 function GateEdit() {
+	const newspack_memberships_gate = window.newspack_memberships_gate || {};
```

Comment:
Even though this file is currently shared between the two strategies, this will likely remain the Memberships approach, and something a lot simpler will handle the gate layout. WDYT of moving this to `memberships-gate-editor.js` for now?